resources/config/config.yaml

```diff
@@ -32,19 +32,20 @@ threads:
   megahit: 8
 
 mem_mb:
-  spades: 10000
-  megahit: 10000
+  spades: 256000
```

Comment:
Generally, it's good form to restrict a PR to just the changes specific to the change you're making. I would usually submit these small fixes as a separate, small PR.